### ISSUE TITLE
Image classification related questions

### ISSUE THREAD

**Issue by saraswat40** [AlexeyAB/darknet#6917]
I am trying to train a set of images as described [here](https://pjreddie.com/darknet/train-cifar/). 
1. For negative examples I simply created another class called background and passed my images as 0_background, 1_backround etc. Can someone confirm that this is correct?
2. I don't know how to generate an f1 score on the test data. In object detection there is a 'map' function available. but this doesn't work with image classification. Can someone tell me how to generate the f1 score?

Thanks

**Comment by stephanecharette** [comment]
No, for negative images you do not want to create a class.  Instead, the label file should be completely empty.

**Comment by saraswat40** [comment]
@stephanecharette I am assuming you mean the file that normally contains the coordinates. This file is needed for object detection only, not image classification.

**Comment by lizq-git** [comment]
Hi, I am currently trying this as well.

So, I assumed that you want to have an extra to keep track of the "not interested" objects.

What I did was:

at the labels.txt, create one class called "OTHERS"

The images related to this class named "x_OTHERS.png/jpg", where x is an integer.

I did able to get a very low loss with the given cifar.cfg on my dataset, but once I try my dataset with the VGG16.cfg it failed badly, the loss became inf after 250ish iteration.

**Comment by saraswat40** [comment]
@lizq-git try another model. I have not tried this one but csresnext50 is working for me.

**Comment by lizq-git** [comment]
> @lizq-git try another model. I have not tried this one but csresnext50 is working for me.

Thanks a lot, will try it now!
